modules/social_features/social_core/social_core.install

```diff
@@ -1366,3 +1366,17 @@ function social_core_update_10105() {
     $config->save();
   }
 }
+
+/**
+ * Add translation support for custom blocks fields.
+ */
+function social_core_update_10200() {
```

Comment:
Since we add this to 10.3 milestone, need to rename the number to start with 103

Reply:
fixed